app/controllers/users_controller.rb

```diff
@@ -1,16 +1,26 @@
 class UsersController < ApplicationController
+  before_action :find_user, only: %i[show edit update]
+  before_action :logged_in_user, only: %i[index edit update destroy]
```

Comment:
nhiều action quá thì chuyển sang dùng except cho gọn nha em
```suggestion
  before_action :logged_in_user, except: %i(show new create)
```